### PR TITLE
making the new contributors profile automatically added

### PR DESCRIPTION
**Pull request by SumitKumar-17**
In the file .github/workflows/github_contributors_automatically_adding.yml  

We need to replace the name of the branch on Line No.4.
I have tested the code by replacing the name on my branch, and the contributors are automatically added, and the issue is fixed .

I have currently replaced the branch name with the "main" branch name.
Please have a Look into the file

### CODE REVIEW

**Comment by abhay-raj19** [comment]
can you show me the video recording of that local test that you made on a separate branch .

**Comment by SumitKumar-17** [comment]
https://github.com/abhay-raj19/FitBody/assets/118801141/6ca383d4-3aa7-460f-9070-ffb6d4c8944f

**Comment by SumitKumar-17** [comment]
Currently what i did as I made the yml file with branch name as  "main" so I replaced the branch name as my testing branch name  and after changing you can see that my contributors is getting automatically added as per the requirement needed by this iisue.

**Comment by abhay-raj19** [comment]
This video is not opening up

**Comment by abhay-raj19** [comment]
Do Join Discord  community and we can discuss all specific changes there and get good reviews 
Link : README.md

**Comment by abhay-raj19** [comment]
That's cool bro , just make one change also that the style of the profile should look some what in circular fashion , some what like this 
image

and maintain the no. of profiles in a row according to what looks good !!

will merge it ASAP 

Thanks of your plenty of the contributions 💙.

**Comment by SumitKumar-17** [comment]
@abhay-raj19 Okay i will start working on the change of circular profile

**Comment by SumitKumar-17** [comment]
@abhay-raj19  i have completed it as per the design and it is done 
Screenshot_2024-05-15_09_32_42

**Comment by abhay-raj19** [comment]
That looks amazing now @SumitKumar-17

File: README.md [hunk 1]
```diff
@@ -144,110 +144,10 @@ To get latest updates on the project and to interact with the community. We are
...
-      <a href="https://github.com/pranshu-5123">
-        <img src="https://github.com/pranshu-5123.png" width="100px;" alt="username2"/><br />
-        <sub><b>Pranshu</b></sub>
-      </a>
-      </td>
-  </tr>
-</table>
-
+<a href="https://github.com/abhay-raj19/FitBody/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=abhay-raj19/FitBody" />
```
Comment (abhay-raj19) [inline README.md:149]: but this image that is embedded automatically generates the image for the contribuors section and then what is the need for adding that workflow in the github/workflow :(

Comment (SumitKumar-17) [reply README.md:149]: @abhay-raj19 

yes, That method was to add contributor profiles by defining the workflow of the yml file. But  using that code it only it makes a rectangular profile . so I changed the way of implementing. 
ya that was my mistake I have deleted the workflow.yml files now.
please have a look on it now

**Comment by abhay-raj19** [comment]
now it is fine though

**Pull request merged**